### PR TITLE
Release announcement for 2.2.1

### PR DESCRIPTION
**Pull request by dobey**

### CODE REVIEW

**Comment by KAMiKAZOW** [comment]
An up-to-date screenshot, perhaps even two – perhaps with Plasma Desktop and Plasma Mobile on Pinephone or whatever – would be great. The phone one could also replace the ancient Ubuntu Touch one.

File: _posts/2022-04-05-maliit-release-2.2.1.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## What is it?
+
+Maliit provides a flexible and cross-platform input method framework. It has a
+plugin-based client-server architecture where applications act as clients and
+communicate with the Maliit server via input context plugins. Maliit is an open
+source framework (LGPL 2.1 or later) and an open source on-screen keyboard (LGPL 3.0 only).
+
+Maliit is being used by [KDE for Plasma](https://blog.martin-graesslin.com/blog/2021/03/using-maliit-keyboard-in-a-plasma-wayland-session/),
+[LG for webOS](https://github.com/webosose/maliit-framework-webos),
```
Comment (KAMiKAZOW) [inline _posts/2022-04-05-maliit-release-2.2.1.md:47]: Does anybody know if Steam Deck's on-screen keyboard is Maliit-based?

Comment (dobey) [reply _posts/2022-04-05-maliit-release-2.2.1.md:47]: I'm pretty sure the virtual keyboard in Game Scope on the deck is an internal thing and not based on Maliit.

Comment (bhush9) [reply _posts/2022-04-05-maliit-release-2.2.1.md:47]: Yes, steam deck uses the keyboard that comes with steam client itself and not maliit.

File: _posts/2022-04-05-maliit-release-2.2.1.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+[Ubuntu Touch](https://gitlab.com/ubports/core/lomiri-keyboard),
+[LuneOS](https://github.com/webOS-ports/webos-keyboard),
+[Sailfish OS](https://git.sailfishos.org/mer-core/maliit-framework),
```
Comment (pvuorela) [inline _posts/2022-04-05-maliit-release-2.2.1.md]: This is nowadays in https://github.com/sailfishos/maliit-framework

**Comment by dobey** [comment]
> An up-to-date screenshot, perhaps even two – perhaps with Plasma Desktop and Plasma Mobile on Pinephone or whatever – would be great. The phone one could also replace the ancient Ubuntu Touch one.

I've replaced the N9 screenshot with one from pinephone. As for adding others, the "screenshots, videos, etc" link is broken, so I've just removed that line of text with multiple broken links. If we want to add a gallery, we should do it in a separate PR which adds a gallery page, and the screenshots, I think.

**Pull request merged**